WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -220,7 +220,32 @@
     <% end %>
 
     <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| c[:name]  }, value_method: lambda { |c| c[:iso_code] }  %>
-    <%= f.input :base_entry_fee_lowest_denomination %>
+    <%= f.input :base_entry_fee_lowest_denomination, input_html: { type: "text" } %>
```

Comment:
Do you think we could implement all of this logic as a new custom input type? You can see the ones we've created already here: https://github.com/thewca/worldcubeassociation.org/tree/master/WcaOnRails/app/inputs. That should actually be easy to do, the real work would be moving the javascript below into application.js.

This is a worthwhile investment, because we're definitely going to be asking the competition organizer for per event prices eventually.